allofplos/plos_regex.py

```diff
@@ -44,8 +43,10 @@ def validate_doi(doi):
 
 def validate_filename(filename):
     """
-    For an individual string, tests whether the full string is in a valid article file in corpusdir or newarticledir
-    format or not.
+    For an individual string, tests whether the full string is in a valid article file. This can take two forms.
+    
+    TODO: Officially document these two forms and give them names. Also, Explain the example below.
```

Comment:
This fails because there are too many digits after `journal.pbio.`. That's the logic already laid out in the DOI validation though. Bet we could use a better False example

Reply:
We could also use a formal spec in the docs that says what the actual different formats are… I don't know if we should really have a single pair of examples here as opposed to a few examples in its own doc page about said spec.